solutions/rev.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Secret is reversed in puzzle/rev.txt. Can you decode it?
+
+rev puzzle/base64.txt
```

Comment:
Uh, that's not the right file.

```suggestion
rev puzzle/rev.txt
```